CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 3.8.2 (September 22, 2020)
+* Improvement: Added a11y improvements
+* Removed unused webRequestBlocking extension permission
```

Comment:
Info about switching to old clipper logo should also be given here?

Reply:
I agree to your point, since the new logo never went to prod, there is no point of adding this info here.